Review _Contribute to What's new or release notes_ page (#490)
grafana/writers-toolkit: docs/sources/contribute-documentation/contribute-release-notes/index.md

---
aliases:
  - /docs/writers-toolkit/writing-guide/contribute-release-notes
date: 2024-02-14
description: This section describes the different ways of contributing to the What's new document or release notes.
keywords:
  - what's new
  - release notes
menuTitle: Contribute to What's new or release notes
title: Contribute to What's new or release notes
weight: 250
---

<!-- vale Grafana.Timeless = NO -->

# Contribute to What's new or release notes

This topic explains the decisions and actions associated with collecting, writing, and publishing What's new content.

{{< admonition type="note" >}}
This topic is only relevant for internal Grafana Labs contributors.
{{< /admonition >}}

## What's new documentation development process

> You publish What’s new content to the website through the website Content Management System (CMS).

Reviewer: While I get changing this out of passive voice, I don't think it improves the sentence or makes it easier to understand. And not using passive voice isn't a hard rule. Sometimes passive voice makes more sense and is easier to understand. I'd stick with the passive voice on this first sentence.
```suggestion
What’s new content is published to the website through the website Content Management System (CMS).
```

To add a new note, browse to the [What's new CMS collection](https://admin.grafana.com/content-admin/#/collections/whats-new).

Because this platform is meant to be used by the entire organization, by default anyone can contribute and publish to What’s new, without the need for approval.

> Quality assurance is a conversation within and between contributing teams and internal stakeholders, but there are some best practice guidelines described in the last two sections of this topic.

Reviewer: ```suggestion
_Quality assurance is a conversation within and between contributing teams and internal stakeholders_, but there are some best practice guidelines described in the last two sections of this topic.
```

> You should enter release notes into the CMS two to four weeks before the feature is available, depending on the size of the product or feature.

Reviewer: ```suggestion
Enter release notes into the CMS two to four weeks before the feature is available, depending on the size of the product or feature.
```

This gives the Go To Market (GTM) team time for promotion and enablement.
For Grafana versioned releases, have your content entered in the CMS by the cut-off date communicated by the delivery team.
For more information, refer to the [Record Announce Document Ship (RADS) guidelines](https://grafana-intranet--simpplr.vf.force.com/apex/simpplr__app?u=/site/a145f000001dCXBAA2/page/a125f000004oOF7AAM).

> {{< admonition type="note" >}}

Reviewer: I know I used the word "note" here, but I didn't intend for this to be in a note, because I don't want folks to skip this as something that's an aside that's not integral to the content. It's integral that folks understand this concept, so it should be part of the body text. We can use a different lead in like maybe "It's important to understand that..."

Author: Ah yeah, I was mostly copying the feedback I got from Clayton in https://github.com/grafana/writers-toolkit/pull/461#discussion_r1480346185.

I'll make the change to use a different lead.

Author: Done in [`1f68d0d` (#490)](https://github.com/grafana/writers-toolkit/pull/490/commits/1f68d0db9aa1156af345901c2db8972e0357dc39).

In the context of the CMS, the word "published" has a slightly different meaning than in general use:

- **Published**: Your entry is complete and in **Published** status.
  It's either visible on the external _What's new in Cloud_ page or is going to automatically become visible on the release date.
- **Live**: Your entry is visible on the _What's new in Cloud_ page.

{{< /admonition >}}

### Create a What's new entry

When you’re ready to add a What’s new entry, complete the following steps:

1. Fill out the fields:

   - **FEATURE NAME**: Short headline for the feature.

>      For example, _Grafana OnCall integration for Alerting_.

Reviewer: With all of these new second lines of content indented at the same level as the parent bullet, the list is less scannable for the field names. It starts to have a "wall of text" feeling for me. Can we indent these second lines?
![image](https://github.com/grafana/writers-toolkit/assets/76437239/a4a66c4a-86f3-4216-b8b0-9fa4fb143a59)

Author: Yeah, it does feel that way, although I think it's unrelated to my changes. It looks quite wall-y already to me.

Maybe we want a description list instead, I'll try that.

Reviewer: Okay leave it. I don't have more suggestions.

Reviewer 2: Maybe a two column table? That's about the only other way I can think of within the limits of markdown...

Author: I was also thinking of that, I've got the commit in jdb/2024-02-review-contribute-to-whats-new-table.

<img width="786" alt="Screenshot 2024-02-15 at 09 22 15" src="https://github.com/grafana/writers-toolkit/assets/4599384/f25261ad-be2a-4dbf-bdea-5a2cae58794b">

Personally, it's my favorite so far.

Reviewer: yeah that's dope!

Reviewer 2: Three... I meant a three column table :-)

Yea that's a LOT easier to read.

     Add any images and a link to your public YouTube video here.
     If you need more information on adding an image, refer to [Image, diagram, and screenshot guidelines](https://grafana.com/docs/writers-toolkit/write/image-guidelines/).

   - **DOCUMENTATION URL (OPTIONAL)**: URL to the public documentation for this feature.
   - **ENABLEMENT VIDEO (OPTIONAL)**: Link to the video used for enablement.
   - **INTERNAL INFORMATION (OPTIONAL)**: Information for Grafana Labs employees only.

     For example, ProductDNA, Slack channel, FAQ, training documentation, or videos.
     Used for training and internal announcements.
     This is only visible on the [internal What's new page](https://admin.grafana.com/whats-new/), _not_ the [public What's New page](https://grafana.com/docs/grafana-cloud/whats-new/).

1. Click **Save**.
   The entry is now in **Draft** status and the CMS opens a pull request in the `grafana/website` repository.
1. If your entry is ready to publish, proceed to step 4.
   If your entry requires review, follow these steps:

   1. In the **Status** drop-down, select **In review.**

>       {{< admonition type="note" >}}

Reviewer: This looks terrible as an admonition. I vote we just leave it as is:
![image](https://github.com/grafana/writers-toolkit/assets/76437239/f9c7a3da-ce55-43f4-aa0d-67d5ebd95797)

Author: That's actually a bug with the admonition shortcode. I've opened https://github.com/grafana/website/pull/18284

>       The Documentation Team doesn't automatically review these pull requests; teams that create What’s new entries are responsible for determining their own review process.

Reviewer: ```suggestion
      _The Documentation Team doesn't automatically review these pull requests; teams that create What’s new entries are responsible for determining their own review process._
```

1. To publish your entry from the CMS, follow these steps:

   1. In the **Status** drop-down, click **Ready**.
   1. In the **Publish** drop-down, click **Publish now**.
      The entry appears in [What's new in Cloud](https://grafana.com/docs/grafana-cloud/whats-new/) on the date you entered.

> For Grafana versioned releases, the content entered in the CMS is published in the versioned What’s new at a later date.

Reviewer: ```suggestion
For Grafana versioned releases, the content you enter in the CMS is published in the versioned What’s new at a later date.
```
